.gitmodules

```diff
@@ -3,3 +3,6 @@
 	url = git@github.com:fmihpc/fsgrid.git
 
 
+[submodule "submodules/dccrg"]
```

Comment:
So how does this work? I don't see where it specifies that it should land on the `vlasiator-version` branch.